### PR TITLE
chore: add icons.json file

### PR DESCRIPTION
**Pull request by aslilac**
Leading up to #8344 

We need a way for the app to know all of the bundled icons so that we can better expose them throughout the UI

### CODE REVIEW

**Review by BrunoQuaresma — APPROVED** [review]
I really liked the idea 😃

**Review by kylecarbs — APPROVED** [review]
LGTM.

Can we add these to our emoji-picker? We also may want to import the `icons.json` file from JS so it's embedded rather than have to make a network request for it.

**Review by matifali — APPROVED** [review]
Great

**Comment by aslilac** [comment]
> Can we add these to our emoji-picker?

Yes!

> We also may want to import the `icons.json` file from JS so it's embedded rather than have to make a network request for it.

Good point! I'll generate it somewhere in src/ instead

File: .gitignore [hunk 1]
```diff
@@ -29,6 +29,7 @@ site/e2e/test-results/*
 site/e2e/states/*.json
 site/e2e/.auth.json
 site/playwright-report/*
+site/static/icons.json
```
Comment (matifali) [inline .gitignore:32]: Why are we adding it ti `.gitignore`?

Comment (aslilac) [reply .gitignore]: I guess it's just habit to ignore generated files

**Pull request merged**